docs/auditors/mounts.md

```diff
@@ -10,9 +10,9 @@ kubeaudit mounts [flags]
 
 ### Flags
 
-| Short   | Long      | Description                                                          | Default                                                                  |
-| :------ | :-------- | :------------------------------------------------------------------- | :----------------------------------------------------------------------- |
-| -s      | --paths   | List of sensitive paths that shouldn't be mounted.                   | [default sensitive host paths list](#Default-sensitive-host-paths-list)  |
+| Short   | Long              | Description                                                          | Default                                                                  |
+| :------ | :---------------- | :------------------------------------------------------------------- | :----------------------------------------------------------------------- |
+| -s      | --denyPathsList   | List of sensitive paths that shouldn't be mounted.                   | [default sensitive host paths list](#Default-sensitive-host-paths-list)  |
```

Comment:
I believe we can go for `deny-paths-list` so the flag has the same format as [the others](https://github.com/Shopify/kubeaudit/blob/8be8c73b59a5cac5d1e5178d18f721838ba6b119/cmd/commands/capabilities.go#L12)

Reply:
Oh yes good catch. Also, I was wondering, do we replace `-s` by `-d` for the `deny` part of the argument?

Reply:
I guess it'd be a good idea to have `-d` as a shorthand, @jcbbc!